terraform/README.md

```diff
@@ -0,0 +1,43 @@
+# How to create Static-Website-Hosting with the help of terraform.
+## Pre Requisites.
+1. AWS account
+2. Terraform 
+3. VS code
+4. Index.html file
+
+# Deployment Steps.
+
+Create a directory.
+
+`mkdir directory name`
+
+Now get into the directory. Use this command.
+
+`cd directory name`
+
+Now you need to create some files in this folder. use this command.
+
+`code filename.tf`
```

Comment:
What is this command?

Reply:
We'll use this command to open our text editer when we type e.g. "code s3 static file.tf" then our text editer will open. we can also use touch command but it is not working with VS code.